client/src/components/hero/Hero.tsx

```diff
@@ -95,24 +212,61 @@ export const Hero = withTheme(styled(UnstyledHero)`
       background-color: #2f4453;
       max-width: 263px;
       max-height: 110px;
+      cursor: pointer;
     }
   }
 
   .hero-text {
     color: white;
     font-weight: 700;
+    word-break: break-word;
+    width: 30rem;
 
     .hero-text-large {
-      font-size: 40px;
+      font-size: 1.75rem;
     }
 
     .hero-text-medium {
-      font-size: 20px;
+      font-size: 1rem;
       line-height: 1;
     }
 
     .hero-text-small {
-      font-size: 9px;
+      font-size: 0.75rem;
     }
   }
+
+  .hero-more {
+    height: 25px;
+    cursor: pointer;
+  }
+
+  .hero-main-image {
+    width: 40rem;
+    height: 23.75rem;
+  }
+
+  .hero-sub-image {
+    width: 10rem;
+    height: 6.875rem;
+    cursor: pointer;
+  }
+
+  .chevron-left {
+    font-size: 5rem;
+    left: 13rem;
+    top: 17rem;
+    position: absolute;
+    color: white;
+    cursor: pointer;
+  }
+
+  .chevron-right {
+    font-size: 5rem;
+    right: 15rem;
+    top: 17rem;
+    position: absolute;
+    color: white;
+    cursor: pointer;
```

Comment:
Do these absolute positioning work well if you shrink the website a bit?

Reply:
removed position absolute and made them relative because the positioning gets messed up.